config.js

```diff
@@ -117,5 +117,26 @@ export const ALL_NEEDED_MODEL_RESOURCES = {
       "https://huggingface.co/Xenova/Phi-3-mini-4k-instruct_fp16/resolve/main/",
     localFolderPathPrefix: "Xenova/",
     resources: ["onnx/model_q4.onnx", "onnx/model_q4.onnx_data"]
+  },
+
+  // ort web wasm
+  "ort-web@1_18_0": {
+    linkPathPrefix: "https://cdn.jsdelivr.net/npm/onnxruntime-web@1.18.0/dist/",
+    localFolderPathPrefix: "frameworks/ort-web/",
+    resources: ["ort-wasm-simd.jsep.wasm"]
+  },
+
+  "ort-web@1_17_1": {
```

Comment:
Can those 1.17.1 demos use 1.18 directly? Keep simpler ONNX Runtime Web dist is better. @NingW101

Reply:
> Keep simpler ONNX Runtime Web dist is better.

Exactly, I remember that I tested but failed before, let me double check it.

Reply:
I tried the samples with transformers which depends on `ort-web-1.18` but failed to run, I tried to debug and find that the root cause was the implementation inside transformers, I guess `transformers-v3` has not covered yet because it is a beta version.
So maybe we still need to keep these 2 wasm version for now.

Reply:
Transformers.js V3 was just out officially: https://x.com/xenovacom/status/1823026862647656515

Reply:
> Transformers.js V3 was just out officially: https://x.com/xenovacom/status/1823026862647656515

Thanks for the info @gyagp, I notice that the package name changed from `@xenova/transformers -> @huggingface/transformers`, `ort-wasm` used by v3 also change to `1.19-dev` and I need to check the if api calls have changed.

@ibelem I'd like to upgrade transformers to v3 in another PR if the everything goes well, since this PR is only for removing the binaries.

Reply:
Thanks @NingW101 Please help to verify if any demos can be upgraded to @huggingface/transformers v3 in next PRs then, merge this PR.